github/projects.go

```diff
@@ -79,28 +79,25 @@ type ListProjectsOptions struct {
 //
 // GitHub API docs: https://docs.github.com/rest/projects/fields
 type ProjectV2FieldOption struct {
-	ID string `json:"id,omitempty"`
-	// The display name of the option.
-	Name string `json:"name,omitempty"`
-	// The color associated with this option (e.g., "blue", "red").
-	Color string `json:"color,omitempty"`
-	// An optional description for this option.
-	Description string `json:"description,omitempty"`
+	ID          *string `json:"id,omitempty"`          // The unique identifier for this option.
+	Name        *string `json:"name,omitempty"`        // The display name of the option.
+	Color       *string `json:"color,omitempty"`       // The color associated with this option (e.g., "blue", "red").
+	Description *string `json:"description,omitempty"` // An optional description for this option.
 }
 
 // ProjectV2Field represents a field in a GitHub Projects V2 project.
 // Fields define the structure and data types for project items.
 //
 // GitHub API docs: https://docs.github.com/rest/projects/fields
 type ProjectV2Field struct {
-	ID        *int64     `json:"id,omitempty"`
-	NodeID    string     `json:"node_id,omitempty"`
-	Name      string     `json:"name,omitempty"`
-	DataType  string     `json:"dataType,omitempty"`
-	URL       string     `json:"url,omitempty"`
-	Options   []*any     `json:"options,omitempty"`
-	CreatedAt *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt *Timestamp `json:"updated_at,omitempty"`
+	ID        *int64                  `json:"id,omitempty"`
+	NodeID    string                  `json:"node_id,omitempty"`
+	Name      string                  `json:"name,omitempty"`
+	DataType  string                  `json:"dataType,omitempty"`
+	URL       string                  `json:"url,omitempty"`
```

Comment:
I know you didn't write these, but optional fields should be reference types.

Reply:
Thanks for catching these @gmlewis I've updated all in a similar state in 31e34f0fa7ce5f587db4d25bd6dbe44400b963f8